triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerLogContext.scala

```diff
@@ -78,14 +98,18 @@ final class TriggerLogContext private (
     val groupSpans = contexts.foldLeft(span) { case (span, context) =>
       span.groupWith(context.span)
     }
+    val context = new TriggerLogContext(loggingContext, groupEntries.toSeq, groupSpans)
+
+    context.setCallback(callback)
```

Comment:
We need to ensure all newly created logging contexts have the current callback defined upon them